orgbook-service/Program.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+
+namespace Gov.Lclb.Cllb.OrgbookService
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = CreateWebHostBuilder(args).Build();
+            host.Run();
+        }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((hostingContext, config) =>
+                {
+                    config.AddEnvironmentVariables();
+                })
+                .ConfigureKestrel(options =>
+                {
+                    options.ListenLocalhost(5023, o => o.Protocols = HttpProtocols.Http2);
```

Comment:
Would prefer not to have this line. How else can I force http2 on all ports?

Reply:
The BC Government OpenShift does not seem to support HTTP 2.0 for health checks.  If you don't have health checks then the deployment can be unreliable; thus you should configure Kestrel to have a HTTP2 port that you will use for GRPC, and a HTTP1 port that you will use for health checks.  The FileManagerService Startup should have an example of this.